### PR TITLE
Add 1.8.1 olm catalog

### PR DESCRIPTION
**Pull request by pure-jliao**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** (optional)
Closes #

**Special notes for your reviewer**:

### CODE REVIEW

File: deploy/olm-catalog/portworx/1.8.1/portworx-certified.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,540 @@
...
+          },
+          "spec": {}
+        }
+      ]
+spec:
+  displayName: Portworx Enterprise
+  version: 1.8.1
+  minKubeVersion: "1.12.0"
+  maturity: stable
+  replaces: portworx-operator.v1.7.1
```
Comment (piyush-nimbalkar) [inline deploy/olm-catalog/portworx/1.8.1/portworx-certified.clusterserviceversion.yaml:55]: This should be 1.8.0

File: test/integration_test/utils/px_operator.go [hunk 2]
```diff
@@ -10,7 +10,7 @@ import (
 )
 
 const (
-	nextReleaseTag = "1.9.0-dev"
+	nextReleaseTag = "1.8.1-dev"
```
Comment (piyush-nimbalkar) [inline test/integration_test/utils/px_operator.go:13]: Should we leave this to 1.9.0 if 1.8.1 is going to be a quick hot fix release?

**Review by piyush-nimbalkar — APPROVED** [review]

File: test/integration_test/utils/px_operator.go (hunk 2 above)
Comment (pure-jliao) [reply test/integration_test/utils/px_operator.go:13]: this doesn't matter, the pipeline will use the default jenkins env parameter, I can update this again once 1.8.1 is released.

File: deploy/olm-catalog/portworx/1.8.1/portworx-certified.clusterserviceversion.yaml (hunk 1 above)
Comment (pure-jliao) [reply deploy/olm-catalog/portworx/1.8.1/portworx-certified.clusterserviceversion.yaml:55]: ok so the rule applies between patch versions as well not only the minor versions?

Comment (piyush-nimbalkar) [reply deploy/olm-catalog/portworx/1.8.1/portworx-certified.clusterserviceversion.yaml]: Yes, all versions

**Pull request merged**